extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -48,11 +48,6 @@ require([
 			event.preventDefault();
 			openCommunityModal(tabs.TAB_ADMINS);
 		});
-
-		$('#viewAllMembers').click(function (event) {
-			event.preventDefault();
-			openCommunityModal(tabs.TAB_ALL);
-		});
```

Comment:
WHY LIKE THIS? this was used to display all contributors on this wiki and has been removed in the new design, to there's no need to keep it.